sklearn/neighbors/_nearest_centroid.py

```diff
@@ -68,6 +85,21 @@ class is the arithmetic mean, which minimizes the sum of squared L1 distances.
 
         .. versionadded:: 1.0
 
+    deviations_ : ndarray of shape (n_classes, n_features)
+        Deviation of each class using soft thresholding.
```

Comment:
this requires more explanation, not easy for a user to understand this from this text.